supertokens_python/recipe/userroles/permission_claim.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+#
+# This software is licensed under the Apache License, Version 2.0 (the
+# "License") as published by the Apache Software Foundation.
+#
+# You may not use this file except in compliance with the License. You may
+# obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+from typing import List, Any, Set, Dict
+
+from supertokens_python.recipe.session.claim_base_classes.primitive_array_claim import (
+    PrimitiveArrayClaim,
+)
+from supertokens_python.recipe.userroles import UserRolesRecipe
+from supertokens_python.recipe.userroles.interfaces import GetPermissionsForRoleOkResult
+
+
+class PermissionClaimClass(PrimitiveArrayClaim[List[str]]):
```

Comment:
why does PrimitiveArrayClaim need to take List[str]? And not just str? Cause the name has Array in it..

Reply:
That's just the way I designed the generics for now. There were issues with directly passing str. I'll try again.

Reply:
This seems complicated. SessionClaim expects a single Generic var via `[...]` (I'm passing PrimitiveList) and also that overrides should return a jsonPrimitive (not a list of jsonPrimitive).